src_py/sprite.py

```diff
@@ -256,7 +256,8 @@ def __init__(self, *groups):
         self.blendmode = 0  # pygame 1.8, referred to as special_flags in
                             # the documentation of Surface.blit
         self._visible = 1
-        self._layer = 0     # READ ONLY by LayeredUpdates or LayeredDirty
+        self._layer = getattr(self, '_layer', 0)    # Default 0 unless 
+                                                    # initialized differently.
```

Comment:
If it should be possible to define `_layer` by the user at instantiation of the **DirtySprit** (or any subclass), then imho, we should do : 

```python

class DirtySprite(Sprite):

    def __init__(self, *groups, layer=0):
        ...
        self._layer = layer
```
And update the documentation of the constructor.
So it clearer, saves one line, and some nice cases, you won't even need an extra subclass...